docs/src/components/color-theme-picker.tsx

```diff
@@ -1,6 +1,7 @@
 import {Box} from '@primer/react'
 import React from 'react'
 import {sentenceCase} from 'sentence-case'
+// eslint-disable-next-line import/no-unresolved
```

Comment:
If someone is attempting to run docs, I think we should run the build as a prerequisite given it's a dependency and avoid suppressing errors. ESLint is technically correct in warning the user about this here imo.

Reply:
As discussed, we do run build before storybook and docs.